### PR TITLE
PixFuture Bid Adapter: redisigned user sync

### PR DESCRIPTION
**Pull request by PolishchukPixFuture**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [x] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
 - Added GVLID to be compatible with TCF v2.0.
 - Added image user sync type.
 - Added COPPA and US Privacy support
 - Refactoring of User Sync module

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[
+    "pixfutureBidAdapter",
+    "appnexusBidAdapter",
+    "consentManagement",
+    "gdprEnforcement"
+]
```
Comment (ChrisHuie) [inline modules.json:6]: This file is for testing and should be removed from pr

File: modules/pixfutureBidAdapter.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
+---
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+---
```
Comment (ChrisHuie) [inline modules/pixfutureBidAdapter.md:6]: These adapter documentation highlights need to go in our docs repo here instead please -> https://github.com/prebid/prebid.github.io

File: modules.json (hunk 1 above)
Comment (pixfuture-media) [reply modules.json]: Hi Chris,

We removed modules.json and added info to https://github.com/prebid/prebid.github.io
Please let us know about our adapter status.

Regards,
Vitali I.

File: modules/pixfutureBidAdapter.md (hunk 2 above)
Comment (PolishchukPixFuture) [reply modules/pixfutureBidAdapter.md]: We have located this information in the docs repo and removed it from the current .md file.

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**